docs/learning/guidelines/release-tags-guidelines.md

```diff
@@ -1,66 +1,205 @@
-# Release Tags
+# Release tags guidelines
 
-Release tags are used to classify API items according to their intended level of support.
-Release tags are included in standard documentation comments and are processed by tools such as [api-extractor](https://api-extractor.com/) to produce an API review file.
-API review files can then be placed under source code control and compared to future SDK versions to ensure that the API is evolving in compatible or expected ways.
+iTwin.js uses [release tags](https://api-extractor.com/pages/tsdoc/doc_comment_syntax/#release-tags) to classify individual APIs according to their intended level of support. Consumers of iTwin.js should consult [API support policies](../api-support-policies.md) for an overview. This article provides iTwin.js contributors with guidance on applying release tags.
 
-## Supported Release Tags
+## Supported tags
 
-The four primary release tags and the "deprecated" tag are described in the [API support policies](../api-support-policies.md). For [Extensions](../frontend/Extensions.md), two additional tags are supported:
+Each API in iTwin.js belongs to one of four API categories as defined by its associated release tag:
 
-- The "extensions" tag indicates that an API is to be included in the `@itwin/core-extension` API. Currently this tag is only useful for APIs in `@itwin/core-frontend` and `@itwin/core-common`; all other packages' APIs can be used directly by extensions. It may become relevant for more packages in the future. A lint rule enforces that the "extensions" tag may only be applied to APIs tagged as "public".
+- `@public` indicates a stable, production-ready API subject to our documented [stability and deprecation policies](../api-support-policies.md).
+- `@beta` indicates an API undergoing stabilization. Users are encouraged to experiment with it in non-production environments.
+- `@alpha` indicates a highly experimental API that users should not yet experiment with.
```

Comment:
Thoughts on this?

"indicates a highly experimental API that is subject to substantial change or removal. Use with caution."

It does not explcitly say do not use or try it out but it does warn of the potential consequences.

Reply:
Reworded in api-support-policies.md.